sdk/src/main/java/com/meniga/sdk/models/budget/CreateBudgetRulesParameters.kt

```diff
@@ -0,0 +1,71 @@
+package com.meniga.sdk.models.budget
+
+import com.meniga.sdk.helpers.MenigaDecimal
+import com.meniga.sdk.models.budget.enums.GenerationType
+import com.meniga.sdk.models.categories.MenigaCategory
+import com.meniga.sdk.webservices.requests.QueryRequestObject
+import com.meniga.sdk.webservices.requests.UpdateBudgetRules
+import org.joda.time.DateTime
+import org.joda.time.Months
+
+data class CreateBudgetRulesParameters(
```

Comment:
Classes are usually not named in the plural form unless they extend a list type.